soperator/soperator_resources.go

```diff
@@ -565,7 +575,7 @@ func (specConfig *SpecConfig) getOperatorRole() *rbacv1.Role {
 			{
 				Verbs:     []string{"get", "list", "watch", "create", "update", "patch", "delete", "deletecollection"},
 				APIGroups: []string{"batch", "extensions"},
-				Resources: []string{"jobs", "cronjobs", "ingresses"},
+				Resources: []string{"jobs", "cronjobs", "ingresses", "endpoints", "namespaces"},
```

Comment:
shouldn't matter, but will need to remove create/delete actions from cluster scope resources in namespaced scope roles